docs/tutorials/articles/sales_dashboard/step_02/step_02.md

```diff
@@ -0,0 +1,198 @@
+---
+hide:
+  - toc
+---
+
+The full code for this step is available 
```

Comment:
We should have the normal download button. Check what is done right now for this tutorial

Reply:
There's a style that you can use for a download button.
Check any code sample in the visual elements documentation pages.

Reply:
Understood, I'm adding it `{: .tp-btn target='blank' }`